pyswmm/__init__.py

```diff
@@ -20,7 +20,7 @@
 VERSION_INFO = (0, 6, 1, 'dev0')
 __version__ = '.'.join(map(str, VERSION_INFO))
 __swmm_version__ = '5.2.1'
-__swmm_sha__ = '854df626b32f74e5cfb3a07fbea3f51b5d7855bb'
```

Comment:
I think this commit is failing on OWA-SWMM because it's missing signature from clahub.

Reply:
@jennwuu it's strange, it looks like it's failing b/c I haven't signed (although I have), but then when you go to the PR, all the checks have passed:
![image](https://user-images.githubusercontent.com/12630578/69627834-6d2b4300-1019-11ea-80f6-377236fd3242.png)
Not sure what's going on there!

Reply:
@bemcdonnell thoughts on why the commit 854df6 seems to be failing on OWA-SWMM?

Reply:
@bemcdonnell @jennwuu I'm not sure why clahub shows as failing for SWMM -- I have signed it.  Any thoughts?

Reply:
This PR looks good to me? All the checks have passed. Hmm

Reply:
@bemcdonnell for some reason clahub is showing as failing in my most recent SWMM commits. But I've signed it so I'm not sure why... I believe that's what held this pyswmm PR up previously, the fact that the OWA-SWMM commit referenced in this version of pyswmm shows as failing (correct me if I'm wrong @jennwuu)

Reply:
Oh sorry, I didn't notice you were talking about SWMM. let me look

Reply:
@bemcdonnell yeah, it shows failed on swmm due missing clahub. really odd, not sure why? Maybe we should try to rebuild this commit?

Reply:
@katmratliff https://www.clahub.com/agreements/OpenWaterAnalytics/Stormwater-Management-Model just to double check, this shows signed for you right?

Reply:
@jennwuu yes it shows as signed for me

Reply:
@bemcdonnell can we bump swmm version to 5.3.0 from 5.3.0.dev0 for SWMM and that can be the new commit # referenced in pyswmm?

Reply:
@bemcdonnell  - poke -

Reply:
@katmratliff, I’m happy with this PR once the OWA SWMM version gets updated. :D